cklib/cklib/utils.py

```diff
@@ -633,3 +630,20 @@ def type_str(o):
     if module == "builtins":
         return cls.__qualname__
     return module + "." + cls.__qualname__
+
+
+class defaultlist(list):
```

Comment:
intended change?

Reply:
Yes, I'm removing an external dependency to a module called `defaultlist` and replace it with a single class of identical function in `cklib.utils`.